### PR TITLE
[autoscaler] Fix worker capping fifo test in new scheduler

### LINKED ISSUE #12435: [New scheduler] Fix test_multi_tenancy::test_worker_capping_fifo

### PR DESCRIPTION
**Pull request by ericl**
In the new scheduler, workers are popped more aggressively. This isn't incorrect, but breaks the test. Fix the test by changing the condition to be a bit more relaxed.

Closes https://github.com/ray-project/ray/issues/12435

### CODE REVIEW

**Review by wuisawesome — APPROVED** [review]

File: python/ray/tests/test_multi_tenancy.py [hunk 1]
```diff
@@ -207,20 +207,20 @@ def foo(x):
     assert len(get_workers()) == 2
 
 
-@pytest.mark.skipif(new_scheduler_enabled(), reason="fails")
 def test_worker_capping_fifo(shutdown_only):
     # Start 2 initial workers by setting num_cpus to 2.
```
Comment (kfstorm) [inline python/ray/tests/test_multi_tenancy.py:211]: Update commment

**Review by kfstorm — APPROVED** [review]
LGTM. One comment.

**Comment by ericl** [comment]
Test still failing

**Comment by kfstorm** [comment]
Still failing

**Comment by ericl** [comment]
Still seems failing due to travis not starting 3 workers when num_cpus=3 somehow. I'm inclined to remove this test since it's inherently brittle, and should be unit tested instead (though not sure if we currently have such a unit test). @kfstorm does this sound ok?

**Comment by kfstorm** [comment]
@ericl It sounds good to me. But can you add a test in `worker_pool_test` in this PR rather than just removing this test?

**Comment by ericl** [comment]
@kfstorm I don't have the bandwidth to do that, we can also ignore the test for now and go ahead with the new scheduler regardless

**Comment by kfstorm** [comment]
@ericl OK, I'll add the test into `worker_pool_test`.

**Pull request merged**